src/topp/FLASHDeconv.cpp

```diff
@@ -354,9 +354,9 @@ class TOPPFLASHDeconv : public TOPPBase
     {
       gradient_rt = std::max(gradient_rt, it.getRT());
       // if forced_ms_level > 0, force MS level of all spectra to 1.
-      if (forced_ms_level > 0)
+      if (forced_ms_level == it.getMSLevel())
```

Comment:
comment doesn't match code

Reply:
> These changes update the MS level of the deconvolved peaks back to that of the input (ie MS level 2), and updates the precursor mass and charge to match the deconvolution.

this sounds slightly hacky... wouldn't it be better to keep the original level already in the deconvolution and add an advanced option to optionally store the output as MS1 for visualization?

Reply:
Good idea. I think that makes sense. I will try to fix like that.